token-lending/program/tests/helpers/flash_loan_receiver.rs

```diff
@@ -43,38 +58,42 @@ impl Processor {
         program_id: &Pubkey,
     ) -> ProgramResult {
         let account_info_iter = &mut accounts.iter();
+        let source_liquidity_token_account_info = next_account_info(account_info_iter)?;
+        let destination_liquidity_token_account_info = next_account_info(account_info_iter)?;
+        let token_program_id = next_account_info(account_info_iter)?;
+        let program_derived_account_info = next_account_info(account_info_iter)?;
 
-        let destination_liquidity_account_info = next_account_info(account_info_iter)?;
-        let repay_token_account_info = next_account_info(account_info_iter)?;
-        let token_program_info = next_account_info(account_info_iter)?;
-        let pda_account_info = next_account_info(account_info_iter)?;
-
-        let token_account = TokenAccount::unpack_from_slice(
-            &destination_liquidity_account_info.try_borrow_mut_data()?,
+        let destination_liquidity_token_account = TokenAccount::unpack_from_slice(
+            &source_liquidity_token_account_info.try_borrow_mut_data()?,
         )?;
-        let (pda, nonce) = Pubkey::find_program_address(&[b"flashloan"], program_id);
+        let (expected_program_derived_account_pubkey, nonce) =
```

Comment:
This is `bump_seed` in `try_find_program_address`

Reply:
Changed `nonce` to `bump_seed`.